packages/legend-query-builder/src/graph/QueryBuilderMetaModelConst.ts

```diff
@@ -142,4 +142,11 @@ export enum QUERY_BUILDER_SUPPORTED_FUNCTIONS {
   OLAP_AVERAGE_RANK = 'meta::pure::functions::math::olap::averageRank',
   OLAP_DENSE_RANK = 'meta::pure::functions::math::olap::denseRank',
   OLAP_ROW_NUMBER = 'meta::pure::functions::math::olap::rowNumber',
+
+  // External Format
```

Comment:
we should use the full path for these and leverage the function we have in query-builder to check if the function name is the function